spring-cloud-loadbalancer/src/main/java/org/springframework/cloud/loadbalancer/blocking/retry/BlockingLoadBalancedRetryFactory.java

```diff
@@ -33,13 +34,31 @@
 
 	private final LoadBalancerProperties loadBalancerProperties;
 
+	private LoadBalancerPropertiesFactory factoryProperties;
```

Comment:
Please rename the field to `propertiesFactory`.

Reply:
Changed